src/App.js

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Router, Route } from 'react-router-dom'
+import { Grommet } from 'grommet'
+import styled from 'styled-components'
+import history from './history'
+import theme from './theme'
+import Home from './screens/Home'
 import './App.css';
 
+const StyledGrommet = styled(Grommet)`
+  height: 100%
+`
+
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router history={history}>
+      <>
```

Comment:
Why a fragment here?

Reply:
Good call. I think that was convention based on things we've done in past repos, but it's not really serving a purpose.